src/main/java/org/eclipse/uprotocol/cloudevent/factory/UCloudEvent.java

```diff
@@ -467,24 +469,14 @@ static CloudEvent fromMessageParts(UUri source, UAttributes attributes, UPayload
     static UPayloadFormat getUPayloadFormatFromContentType(String contentType){
         if(contentType == null)
```

Comment:
want to try using a Stream with a filter.
Nothing wrong with for loops and if statements - just saying.
This method is going to be called a lot - nice if it was efficient.
I take it the unit tests did not change and you ran them all. :)

Reply:
Done please re-review